content/posts/blog/easy-xmpp-challenge.md

```diff
@@ -87,12 +86,39 @@ are a first step in that direction, and getting them into your client today
 will make new users' lives easier. But there is also potential to further
 streamline the whole process.
 
-### Call for Action
+### Easy Group Chats
+
+There are two typical IM use cases for group chats: chat with multiple friends
+at the same time (ad-hoc), or participate in a public chat room (typically for
+support purposes, pioneered by
+[Internet Relay Chat](https://en.wikipedia.org/wiki/Internet_Relay_Chat)).
+
+The latter is well-covered by [MUC](https://xmpp.org/extensions/xep-0045.html),
+and the upcoming [MIX](https://xmpp.org/extensions/xep-0369.html) should
+provide a solid technical basis for both use cases. The next logical step is
+to allow the easy
+[creation and sharing of ad-hoc rooms](https://wiki.xmpp.org/web/Easy_Group_Chats)
+between different clients, with a coherent user experience.
+
+### Fighting Spam
+
+If the amount of spam is a measure of XMPP's popularity, the network is doing
+exceptionally well. Almost all of today's XMPP spam can be blocked with some
+easy pattern matching rules, and the inevitable arms race will move to the
+next round.
+
+The community needs to prepare for that, by improving
+[reporting](https://xmpp.org/extensions/xep-0161.html) between server
```

Comment:
Minor styling nitpick: I prefer to have links more descriptive. Imagine that only the marked up text is read (which may very likely be how accessibility software operates!). I’d extend the link brackets to "improving reporting between server operators".